src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -735,14 +774,19 @@ private function bookmark_tag() {
 	/**
 	 * Returns the uppercase name of the matched tag.
 	 *
+	 * The semantic rules for HTML rename certain tags and reprocesses
+	 * them. Because of this, the tag name presented by the HTML Processor
+	 * may different from the one reported by the HTML Tag Processor,
+	 * which doesn't apply these semantic rules.
```

Comment:
```suggestion
	 * The semantic rules for HTML rename certain tags and reprocess
	 * them. Because of this, the tag name presented by the HTML Processor
	 * may be different from the one reported by the HTML Tag Processor,
	 * which doesn't apply these semantic rules.
```

Reply:
circumlocuted in 2ced2f7d7da5ee7b46bfdaa224e251e0157f008c